.github/workflows/test-install.yml

```diff
@@ -0,0 +1,153 @@
+name: test-install
+
+on:
+  push:
+    branch:
+      - master
+      - develop
+      - release/*
+      - dm/*
+    paths:
+      - 'setup.*'
+      - 'environment.yml'
+      - 'requirements*.txt'
+      - 'pyproject.toml'
+      - '.github/workflows/*'
```

Comment:
Is that even necessary?
I.e. do we need to try to install again, when the github actions change?
maybe just of the actual test-install workflow changes?